### DISCUSSION TITLE
Edge feature predictions?

### DISCUSSION THREAD

**Discussion by awezmm** [stellargraph/stellargraph#1827] [Q&A]
Hi,
Is there some example on the StellarGraph docs that deals with edge features? In particular, is there any example that learns to predict edge features? Let's say you have a graph with nodes, edges, and 5 edge features for every edge. You provide the model with all nodes, all edges, but only 4 out of 5 edge features and have it predict the 5th edge feature. Is there some example of this?

**Comment by CalmLogarithm** [comment]
Short answer: no.

You might want to start experimentally with a non-graphical model with 4 inputs and 1 output. That would give you an idea of baseline performance.

Then, the [GraphSAGE link prediction example](https://stellargraph.readthedocs.io/en/stable/demos/calibration/calibration-link-prediction.html) is the next best place. I'm not entirely sure how to modify the code to include known edge features as inputs. I recently [found a fix](https://github.com/stellargraph/stellargraph/discussions/1826#discussioncomment-135411) for sequences and multiple outputs. You might be able to modify that as part of the solution.
